README.md

```diff
@@ -118,7 +118,12 @@ public class PresidentClassMap : ExcelClassMap<President>
 using (var stream = File.OpenRead("Presidents.xlsx"))
 using (var importer = new ExcelImporter(stream))
 {
+    //You can register class maps by type
     importer.Configuration.RegisterClassMap<PresidentClassMap>();
+    
+    //Or by namespace
+    importer.RegisterMapperClassesByNamespace("NAMESPACE");
```

Comment:
Maybe rename this to be ‘My.Namespace`?

Reply:
Done, do you think it's a good idea to leave My.Namespace without quotes ? So that if someone copy-pastes it he'll immediately see.